Formula/enchant.rb

```diff
@@ -24,11 +24,15 @@ def install
                           "--enable-relocatable"
 
     system "make", "install"
+    if OS.mac?
+      # Use AppleSpell for all languages by default
+      system "echo *:AppleSpell > #{share}/enchant/enchant.ordering"
+    end
```

Comment:
```suggestion
    # Use AppleSpell for all languages by default
    (share/"enchant/enchant.ordering").write "*:AppleSpell\n" if OS.mac?
```

Reply:
This produces
RuntimeError: Will not overwrite /usr/local/Cellar/enchant/2.3.2/share/enchant/enchant.ordering

Reply:
Then you can use `atomic_write` instead of `write`.

That said, if upstream ship a default configuration, I don't think we should be replacing it especially if it results in a regression that makes the current test fail.

What does the current configuration that we're replacing do?

Reply:
The current configuration has the following in the system-wide enchant.ordering:
```
*:hunspell,nuspell,aspell
en_AU:aspell,hunspell,nuspell
en_CA:aspell,hunspell,nuspell
en_GB:aspell,hunspell,nuspell
en_US:aspell,hunspell,nuspell
fi:voikko,hunspell,nuspell,aspell
fi_FI:voikko,hunspell,nuspell,aspell
he:hspell,hunspell,nuspell
he_IL:hspell,hunspell,nuspell
tr:zemberek,nuspell
tr_TR:zemberek,nuspell
```
The user file overrides these settings but an entry for `*` only overrides the first line.  Thus, aspell will be used for en_AU, en_GB, etc. and the automatic language detection of AppleSpell is not available.  If one wants to use AppleSpell for, say, en_US, this entry has to be added to the user file as well and this is difficult to figure out because it is not documented.  I think the advantages of automatic language detection far outweigh the test failure (which seems to be tailored to aspell).